Upstream: No
Source: Tom Briden <tom@decompile.me.uk>
Reason: LDFlags not available until lld-11

diff --git a/build/config/compiler/BUILD.gn.old b/build/config/compiler/BUILD.gn
index d2dfd3d..0f603b6 100644
--- a/build/config/compiler/BUILD.gn.old
+++ b/build/config/compiler/BUILD.gn
@@ -395,14 +395,6 @@ config("compiler") {
         "-Wl,--as-needed",
       ]
     }
-    if (use_lld && !is_chromeos_device) {
-      # TODO(thakis): Fix dump_syms to not need this and then remove it,
-      # https://crbug.com/1105559
-      ldflags += [
-        "-Wl,-z,dead-reloc-in-nonalloc=*=0",
-        "-Wl,-z,dead-reloc-in-nonalloc=.debug_ranges=1",
-      ]
-    }
   }
 
   # Linux-specific compiler flags setup.
